### PR TITLE
get r package versions from archive.

### PR DESCRIPTION
**Pull request by Athanasiamo**

### CODE REVIEW

File: 3rdparty/r_pgks.mk [hunk 1]
```diff
@@ -11,8 +11,8 @@ _all: all
 # note: the non-Archive urls will break over time. this wil need updates
 r_pkg_urls := https://cran.r-project.org/src/contrib/Archive/digest/digest_0.6.25.tar.gz \
 	https://cran.r-project.org/src/contrib/Archive/jsonlite/jsonlite_1.7.1.tar.gz \
-	https://cran.r-project.org/src/contrib/RPostgreSQL_0.6-2.tar.gz \
-	https://cran.r-project.org/src/contrib/DBI_1.1.0.tar.gz
+	https://cran.r-project.org/src/contrib/00Archive/RPostgreSQL/RPostgreSQL_0.6-2.tar.gz \
```
Comment (f-krull) [inline 3rdparty/r_pgks.mk:14]: should be consistent with the jsonlite/digest URLs? - either all with `00` or without

Comment (Athanasiamo) [reply 3rdparty/r_pgks.mk:14]: fair point. Added `00` to digest, as this is what it looks like when browsing the CRAN src direclty

Comment (f-krull) [reply 3rdparty/r_pgks.mk:14]: What about jsonlite?

> as this is what it looks like when browsing the CRAN src direclty

But that sounds as if it was better to remove the 00. There seems to be an alias and without the 00 it looks more official. Your choice.

**Review by f-krull — APPROVED** [review]

File: 3rdparty/r_pgks.mk (hunk 1 above)
Comment (f-krull) [reply 3rdparty/r_pgks.mk:14]: The question basically: `.../Archive/...` or `.../00Archive/...`, which one is more likely to break over time?

Comment (Athanasiamo) [reply 3rdparty/r_pgks.mk:14]: I think 00Archive is less likely to break, as it puts the archive at the top of all the versions and current packages. But I think they are linked to eachother in some way anyway.

Comment (f-krull) [reply 3rdparty/r_pgks.mk:14]: Ahhh 💡 So `00Archive` is a hack to make the directory show up on top in the webserver's directory view.
Now I'm pretty sure `Archive` is the canonical name.

Comment (f-krull) [reply 3rdparty/r_pgks.mk:14]: PR is approved. Owner merges :slightly_smiling_face:

Comment (Athanasiamo) [reply 3rdparty/r_pgks.mk:14]: got stressed about prepping for tomorrow, so forgot it :P Thanks!

**Pull request merged**